scripts/indexes.js

```diff
@@ -447,6 +447,12 @@ db.subscriptions.ensureIndex({
     "owner_type": 1,
     "owner": 1
 })
+db.subscriptions.createIndex({
+    "last_updated": 1,
+}, {
+    sparse: true,
```

Comment:
Does it really work to have a sparse TTL index? I can't tell from the docs.

Reply:
I don't see anything saying that you can't. Creating it in the shell yields no errors:
`> db.foo.createIndex({bar:1}, {sparse:true, expireAfterSeconds: 2})
{
	"createdCollectionAutomatically" : false,
	"numIndexesBefore" : 1,
	"numIndexesAfter" : 2,
	"ok" : 1
}`

Reply:
yeah it does. the TTL code does an index scan.